src/events/google_cloud.js

```diff
@@ -79,6 +87,132 @@ function bigQueryWrapper(wrappedFunction) {
     };
 }
 
+const getLastSplittedString = (array) => {
```

Comment:
getLastSplittedItem (doesn't have to be string). also get the delimiter as a param and maybe move to helpers